bindings/python/mappings.py

```diff
@@ -72,14 +73,35 @@ def instance_routes(
     if isinstance(instances, dlite.Instance):
         instances = [instances]
 
+    # FIXME: Bad workaround for adding missing units to the registry.
+    # Too be fixed ASAP.
+    ureg = quantity(1.0, "m").units._REGISTRY
+    unit_definitions = [
+        ("atom_fraction", "atom_fraction = 1"),
+    ]
+    for uname, udef in unit_definitions:
+        if uname not in ureg:
+            ureg.define(udef)
```

Comment:
I do not understand why we hardcode the addition of a missing unit here. Should we not rather allow for the addition of a unit by the user if needed?

Reply:
Definitely. This just a quick hacky workaround to add atom_fraction to the pint unit registry such that the ss3 test works.

Reply:
I really don't think we should putin quick and dirty fixes like this into the main DLite. How much work is it to find a proper solution?

Reply:
Ok, removed the hack. The price for that is that SS3 will not work...

A proper solution will probably take 6h to implement. It will mainly involve tripper (issue https://github.com/EMMC-ASBL/tripper/issues/154) with some fixes to DLite as well.